apps/src/dance/songs.js

```diff
@@ -87,6 +87,8 @@ export function loadSong(songId, songData, onPreloadError) {
 
 export function unloadSong(songId, songData) {
   const url = songData[songId].url;
+
+  Sounds.getSingleton().stopPlayingURL(url);
```

Comment:
Just curious, what's this change for?

Reply:
This one handles stop playing song when selecting the other one in select

Reply:
Gotcha. Not a necessary change - but I'm wondering if this should happen internally in `unload()`? I'd imagine we'd always want to stop a sound before unloading if it's already playing. But @breville maybe you have more historical context on this code?

Reply:
Yeah, I was thinking same way, but not. Currently you can still have song playing when it's unloaded